artemis/config.py

```diff
@@ -838,6 +838,12 @@ class Nuclei:
                 "NUCLEI_CHUNK_SIZE is 200, three calls will be made with 200 templates each.",
             ] = get_config("NUCLEI_CHUNK_SIZE", default=200, cast=int)
 
+            NUCLEI_RUN_TECH_DETECTION: Annotated[
```

Comment:
I don't think this needs to be configurable - adding too many configuration options can also be a burden (for developers but also for users)